core/src/main/java/org/apache/carbondata/core/datastore/page/ComplexColumnPage.java

```diff
@@ -74,4 +75,70 @@ public void putComplexData(int rowId, int depth, List<byte[]> value) {
   public int getDepth() {
     return depth;
   }
+
+  @Override
+  public void putInt(int rowId, int value) {
```

Comment:
How about add default implementation in ColumnPage and override the one that is needed in ComplexColumnPage